vtex/faststore
# chore(gatsby-theme-store): Remove search related components

## What's the purpose of the PR?
This PR is a continuation of #878 and removes all search related UI components to the stores. 

## Upgrading my store
To upgrade your store, just copy+paste the files I've deleted into your store and you should be good to go

## How can this be tested?
This is an architectural change only and should not impact on the final UI. Make sure this is the case on:
